web/src/OverviewTable.tsx

```diff
@@ -630,10 +636,14 @@ export function Table(props: TableProps) {
       <tbody {...getTableBodyProps()}>
         {page.map((row: Row<RowValues>) => {
           prepareRow(row)
+
+          let rowClasses =
+            (rowIsDisabled(row) ? "isDisabled " : "") +
+            (isSelected(row.original.name) ? "isSelected " : "")
```

Comment:
i keep meaning to pull these nested maps into separate components so it makes conditional logic + semantic markup easier 😅 maybe i'll do that in one of the other ux debt tickets if there's the opportunity!

(this implementation is totally fine! i just have this thought every time i look at this component.)